frame/system/src/lib.rs

```diff
@@ -200,12 +200,52 @@ pub mod pallet {
 	use crate::{self as frame_system, pallet_prelude::*, *};
 	use frame_support::pallet_prelude::*;
 
+	pub mod preludes {
+		use super::*;
+		pub mod testing {
```

Comment:
Unfortunately looks like it can't:

```
non-inline modules in proc macro input are unstable
see issue #54727 <https://github.com/rust-lang/rust/issues/54727>
```